img/posts/2020-06-fee-overpayment-attack.dot

```diff
@@ -0,0 +1,56 @@
+digraph fake_coinjoin {
+
+rankdir=LR
+
+node [ shape = "box" ];
+edge [ style = "invis" ];
+
+subgraph cluster_fake1 {
+  fake1_in_label [ label = "Inputs", shape = "none", height=0.1 ];
+  fake1_out_label [ label = "Outputs", shape = "none", height=0.1 ];
+  fake1_in_mine [ label = "1 BTC", style = "filled", fillcolor = "grey" ]
+  fake1_in_other [ label = "0.0001 BTC" ];
+  fake1_out_mine [ label = "1 BTC", style = "filled", fillcolor = "grey" ];
+  fake1_out_other [ shape = "none", label = "" ];
+  fake1_in_mine -> fake1_out_other;
+  fake1_in_other -> fake1_out_mine;
+
+  label = "Perceieved transaction 1\nCalculated fee: 0.0001 BTC";
```

Comment:
"Perceived" here and line 34 below (and in the generated svg)

Great work on the diagram!